src/Grid.js

```diff
@@ -0,0 +1,184 @@
+/** @jsx jsx */
+import React from 'react';
+import PropTypes from 'prop-types';
+// import { css, cx } from 'react-emotion';
+import { css, jsx } from '@emotion/core';
+import config from './GridConfig';
+
+const {
+  gridMaxWidth,
+  columnNumber,
+  gutterWidth,
+  mobileBreakpoint,
+} = config;
+
+const columnWidthPX = gridMaxWidth / columnNumber;
+const columnWidth = (columnWidthPX / gridMaxWidth) * 100;
```

Comment:
🤔 

I'm not sure that's correct? The first one is a pixel value (hence PX) but the second one is a percentage value, so they're not necessarily always going to be the same number, which is why we need both.

I tried working it out with some really basic math on paper and this is what I got ....

```
Let's say that ... 
max width = 200
number of columns = 5

width in pixels = 200 / 5 = 40 (pixels)
width in % = (40 / 200) x 100 = 20 (percent)

```

Reply:
Update: I messed about with the numbers some more and realised we could also achieve the width in percent value by doing `100 / columnNumber` so I could probably simplify this regardless

Reply:
Right. I'm bad at math. It should be `columnWidth = 100 / columnNumber;`.

That makes sense, because each column takes up an equal percentage, so dividing 100 by the number of columns will effectively split 100% into `columnNumber` equal slices.

But just to prove to myself that I could still pass junior high (elementary?) math, my workings:
```
Key, so that the equation-y stuff is more concise and easy to compare
x = gridMaxWidth
y = columnNumber
z = columnWidthPX = x/y

                z        
columnWidth = (---) * 100
                x

    x
   ---
= ( y ) * 100               replacing z with x / y
   ---
    x

     x
=   ---   * 100             merge divisors
  ( y*x ) 

    1
= (---) * 100               simplify by removing x
    y

   100
= (---)                     Multiply 1/y by 100
    y

= 100 / columnNumber
```